src/sdk/model/vapp-network/__json__/vapp-network-static-route-request-json.ts

```diff
@@ -0,0 +1,5 @@
+export interface VappNetworkStaticRouteRequestJson {
+  name: string;
+  network: string;
+  next_hop_i_p: string;
```

Comment:
@bsnyder788 this looks like bad serialization api side

Reply:
@cfsnyder hum, i think it's just because field is named 'NextHopIP'.. IP is uppercase instead of CamelCase

Reply:
yeah it is, and thats a bug api side that needs to be fixed